### ISSUE TITLE
Composer config --file option does not prevent prompt for using a parent dir's composer.json

### ISSUE THREAD

**Issue by TerrePorter** [composer/composer#8023]
**TLDR: If --file option is given to config command, and there is a composer.json in a parent dir but not CWD, composer should not prompt to use the one in parent dir.**

My `composer.json`:

```json
{
    "require": {
        "sunra/php-simple-html-dom-parser": "^1.5"
    },
    "config": {
        "vendor-dir": "/home/vagrant/Code/phptesting/public/ComposerHandler/c972bed92fa232ed49c9dc08aa83f765/"
    }
}
```

Output of `composer diagnose`:

```
Checking platform settings: OK
Checking git settings: OK
Checking http connectivity to packagist: OK
Checking https connectivity to packagist: OK
Checking github.com rate limit: OK
Checking disk free space: OK
Checking pubkeys:
Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: OK
Composer version: 1.8.4
PHP version: 7.3.1
PHP binary path: /usr/bin/php7.3

```

When I run this command:

```
composer -vvv config --file="/home/vagrant/Code/phptesting/public/testdir/composer.json"
```

I get the following output:

```
No composer.json in current directory, do you want to use the one at /home/vagrant/Code/phptesting/public/? [Y,n]? n
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.1-1+ubuntu18.04.1+deb.sury.org+1 on Linux / 4.15.0-45-generic
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/Code/phptesting/public/testdir/composer.json
```

And I did not expect this to happen:

No composer.json in current directory, do you want to use the one at /home/vagrant/Code/phptesting/public? [Y,n]? n

I specified a composer.json file. Why ask me?

Expanding on the issue. When I run this command:

```
composer install -vvv config --file="/home/vagrant/Code/phptesting/public/testdir/composer.json"
(or update)
```

I get the following output:

```
$ composer install -vvv config --file="/home/vagrant/Code/phptesting/public/testdir/composer.json"
No composer.json in current directory, do you want to use the one at /home/vagrant/Code/phptesting/public? [Y,n]? n
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.1-1+ubuntu18.04.1+deb.sury.org+1 on Linux / 4.15.0-45-generic


  [Symfony\Component\Console\Exception\RuntimeException]
  The "--file" option does not exist.


Exception trace:
 () at phar:///usr/local/bin/composer/vendor/symfony/console/Input/ArgvInput.php:218
 Symfony\Component\Console\Input\ArgvInput->addLongOption() at phar:///usr/local/bin/composer/vendor/symfony/console/Input/ArgvInput.php:152
 Symfony\Component\Console\Input\ArgvInput->parseLongOption() at phar:///usr/local/bin/composer/vendor/symfony/console/Input/ArgvInput.php:82
 Symfony\Component\Console\Input\ArgvInput->parse() at phar:///usr/local/bin/composer/vendor/symfony/console/Input/Input.php:54
 Symfony\Component\Console\Input\Input->bind() at phar:///usr/local/bin/composer/vendor/symfony/console/Command/Command.php:204
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:835
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:185
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/composer/src/Composer/Console/Application.php:258
 Composer\Console\Application->doRun() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/composer/src/Composer/Console/Application.php:104
 Composer\Console\Application->run() at phar:///usr/local/bin/composer/bin/composer:56
 require() at /usr/local/bin/composer:24

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...
```

I expected this to happen:

Use the specified composer.json and install the package.

--

Am i reading the help incorrectly? In the help it says, “--file="..." (-f): Operate on a specific file instead of composer.json. Note that this cannot be used in conjunction with the --global option.” 

It sounds like i can specify the composer.json file I want to use.

**Comment by alcohol** [comment]
Not sure I can reproduce this. Can you be more specific about where you are running the command, and whether or not the file you are passing actually exists? (show the output of a `ls -la <path/to/file>` for example).


```console
rob@thinkpad ~ $ cd $(mktemp -d)
rob@thinkpad /tmp/tmp.nmpqrIgMyf $ mkdir -vp some/nested/directory
mkdir: created directory 'some'
mkdir: created directory 'some/nested'
mkdir: created directory 'some/nested/directory'
rob@thinkpad /tmp/tmp.nmpqrIgMyf $ echo '{}' > some/nested/directory/composer.json
rob@thinkpad /tmp/tmp.nmpqrIgMyf $ composer config -vvv -f some/nested/directory/composer.json platform.php 7.3.0
Loading config file /home/rob/.config/composer/config.json
Loading config file /home/rob/.config/composer/auth.json
Failed to initialize global composer: Composer could not find the config file: /home/rob/.config/composer/composer.json
To initialize a project, please create a composer.json file as described in the https://getcomposer.org/ "Getting Started" section
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.2 on Linux / 4.20.13-arch1-1-ARCH
Loading config file /home/rob/.config/composer/config.json
Loading config file /home/rob/.config/composer/auth.json
Failed to initialize global composer: Composer could not find the config file: /home/rob/.config/composer/composer.json
To initialize a project, please create a composer.json file as described in the https://getcomposer.org/ "Getting Started" section
Loading config file /home/rob/.config/composer/config.json
Loading config file /home/rob/.config/composer/auth.json
Reading some/nested/directory/composer.json
rob@thinkpad /tmp/tmp.nmpqrIgMyf $ cat some/nested/directory/composer.json 
{
    "config": {
        "platform": {
            "php": "7.3.0"
        }
    }
}
```

**Comment by TerrePorter** [comment]
Hi @alcohol 

Thanks for the example code of it working. I did your example test and it worked here also.

I verified everything and all the files existed where i used them and i still got the same error. 

So i moved everything to a new testing directory and for some reason i did not get an error. This didn't make sense as nothing but location of the files had changed. So I put everything back to the original locations and i got the error. 

I started playing around and did some more idea testing and figured out the problem.

I am going to duplicate your example as it is a working starting point.

```
vagrant@hs810c:~$ cd $(mktemp -d)
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ mkdir -vp some/nested/directory
mkdir: created directory 'some'
mkdir: created directory 'some/nested'
mkdir: created directory 'some/nested/directory'
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ mkdir -vp some/nested/directory/diff_composer_json/
mkdir: created directory 'some/nested/directory/diff_composer_json/'
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ echo '{}' > some/nested/directory/diff_composer_json/composer.json
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ composer config -vvv -f some/nested/directory/diff_composer_json/composer.json platform.php 7.3.0
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.1-1+ubuntu18.04.1+deb.sury.org+1 on Linux / 4.15.0-45-generic
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading some/nested/directory/diff_composer_json/composer.json
```

Continuing off your example.


```
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ mkdir -vp some/nested/directory/diff_composer_json/
mkdir: created directory 'some/nested/directory/diff_composer_json/'
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ mv some/nested/directory/composer.json some/nested/directory/diff_composer_json/composer.json
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$  echo '{}' > some/nested/composer.json
vagrant@hs810c:/tmp/tmp.3VMeZHryIN$ cd some/nested/directory/
vagrant@hs810c:/tmp/tmp.3VMeZHryIN/some/nested/directory$ composer -vvv config --file diff_composer_json/composer.json
No composer.json in current directory, do you want to use the one at /tmp/tmp.3VMeZHryIN/some/nested? [Y,n]? n
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.1-1+ubuntu18.04.1+deb.sury.org+1 on Linux / 4.15.0-45-generic
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading diff_composer_json/composer.json
```

So it looks like if composer finds a composer.json file in a parent directory [/tmp/tmp.3VMeZHryIN/some/nested] of where composer was executed [some/nested/directory/], it will produce the prompt to use the found file. 

However, if that parent directory composer.json does not exist then everything works as expected.

```
vagrant@hs810c:/tmp/tmp.3VMeZHryIN/some/nested/directory$ rm ../composer.json
vagrant@hs810c:/tmp/tmp.3VMeZHryIN/some/nested/directory$ composer -vvv config --file diff_composer_json/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.1-1+ubuntu18.04.1+deb.sury.org+1 on Linux / 4.15.0-45-generic
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading diff_composer_json/composer.json
```

Ok, that explains the first issue. The second issue still does not work.


```
vagrant@hs810c:/tmp/tmp.3VMeZHryIN/some/nested/directory$ composer install  -vvv config --file diff_composer_json/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/config.json
Loading config file /home/vagrant/.composer/auth.json
Loading config file /home/vagrant/.composer/composer.json
Loading config file /home/vagrant/.composer/auth.json
Reading /home/vagrant/.composer/auth.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/home/vagrant/.composer): git branch --no-color --no-abbrev -v
Executing command (/home/vagrant/.composer): git describe --exact-match --tags
Executing command (/home/vagrant/.composer): git log --pretty="%H" -n1 HEAD
Executing command (/home/vagrant/.composer): hg branch
Executing command (/home/vagrant/.composer): fossil branch list
Executing command (/home/vagrant/.composer): fossil tag list
Executing command (/home/vagrant/.composer): svn info --xml
Reading /home/vagrant/.composer/vendor/composer/installed.json
Running 1.8.4 (2019-02-11 10:52:10) with PHP 7.3.1-1+ubuntu18.04.1+deb.sury.org+1 on Linux / 4.15.0-45-generic


  [Symfony\Component\Console\Exception\RuntimeException]
  The "--file" option does not exist.


Exception trace:
 () at phar:///usr/local/bin/composer/vendor/symfony/console/Input/ArgvInput.php:218
 Symfony\Component\Console\Input\ArgvInput->addLongOption() at phar:///usr/local/bin/composer/vendor/symfony/console/Input/ArgvInput.php:154
 Symfony\Component\Console\Input\ArgvInput->parseLongOption() at phar:///usr/local/bin/composer/vendor/symfony/console/Input/ArgvInput.php:82
 Symfony\Component\Console\Input\ArgvInput->parse() at phar:///usr/local/bin/composer/vendor/symfony/console/Input/Input.php:54
 Symfony\Component\Console\Input\Input->bind() at phar:///usr/local/bin/composer/vendor/symfony/console/Command/Command.php:204
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:835
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:185
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/composer/src/Composer/Console/Application.php:258
 Composer\Console\Application->doRun() at phar:///usr/local/bin/composer/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/composer/src/Composer/Console/Application.php:104
 Composer\Console\Application->run() at phar:///usr/local/bin/composer/bin/composer:56
 require() at /usr/local/bin/composer:24

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...
```

It suddenly ignores the config file setting.

Thanks again for the assistance.

**Comment by alcohol** [comment]
Actually, your first issue I find more puzzling. You should not have to remove the composer.json in the parent directory. I will double check what is going on there.

Your second issue is rather obvious though, to me. The command you are running is invalid. You cannot combine 'install' and 'config'. They are both different commands. There is no combination of the two.

The `--file` flag is only supported by `config`. This is because config can deal with multiple files, not just `composer.json` (it can also handle the `auth.json` file for example).

I think in your case it makes more sense to use the `COMPOSER` environment variable.

```
/workdir $ COMPOSER=foo/bar/composer.json composer validate
foo/bar/composer.json is valid for simple usage with composer but has
strict errors that make it unable to be published as a package:
See https://getcomposer.org/doc/04-schema.md for details on the schema
name : The property name is required
description : The property description is required
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.

/workdir $ COMPOSER=foo/bar/baz/composer.json composer validate
foo/bar/baz/composer.json is valid for simple usage with composer but has
strict errors that make it unable to be published as a package:
See https://getcomposer.org/doc/04-schema.md for details on the schema
name : The property name is required
description : The property description is required
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.

/workdir $ COMPOSER=foo/bar/foo/composer.json composer validate
foo/bar/foo/composer.json is valid for simple usage with composer but has
strict errors that make it unable to be published as a package:
See https://getcomposer.org/doc/04-schema.md for details on the schema
name : The property name is required
description : The property description is required
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.
/workdir $ cd foo/bar/baz

/workdir/foo/bar/baz $ COMPOSER=../foo/composer.json composer validate
../foo/composer.json is valid for simple usage with composer but has
strict errors that make it unable to be published as a package:
See https://getcomposer.org/doc/04-schema.md for details on the schema
name : The property name is required
description : The property description is required
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.

/workdir/foo/bar/baz $ COMPOSER=../composer.json composer validate
../composer.json is valid for simple usage with composer but has
strict errors that make it unable to be published as a package:
See https://getcomposer.org/doc/04-schema.md for details on the schema
name : The property name is required
description : The property description is required
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.
```

**Comment by TerrePorter** [comment]
"Your second issue is rather obvious though, to me. The command you are running is invalid. You cannot combine 'install' and 'config'. They are both different commands. There is no combination of the two."

That is what i was afraid of. Ok, i will see if i can use the environment variable. Thanks
